scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -929,14 +929,7 @@
   (with-redefs [completion/plugin completion/no-op]
     (let [uri "datomic:mem://test-handle-status-update"
           conn (restore-fresh-database! uri)
-          tasks-killed (atom #{})
```

Comment:
The tasks-killed and driver machinery to update it is dead and unchecked. Shall I open a followup to resurrect it?

Reply:
I'm guessing you mean the machinery in the test is dead? If so, do we have any other tests for this functionality?

Reply:
Correct. Don't know. I suspect not. (at least at the unit test level)